nodes/src/nodes/tool_exa_search/IGlobal.py

```
@@ -0,0 +1,85 @@
+class IGlobal(IGlobalBase):
+    """Global state for tool_exa_search."""
+
+    driver: ExaSearchDriver | None = None
+
+    def beginGlobal(self) -> None:
+        if self.IEndpoint.endpoint.openMode == OPEN_MODE.CONFIG:
+            return
+
+        cfg = Config.getNodeConfig(self.glb.logicalType, self.glb.connConfig)
+
+        apikey = str((cfg.get('apikey') or '')).strip()
```

Ruff (0.15.7)

[warning] Avoid extraneous parentheses

Remove extraneous parentheses

(UP034)